tl

```diff
@@ -787,6 +873,14 @@ function project:files(inc_patt_arr, exc_patt_arr, dirname) -- iterate over the
    end
    return coroutine.wrap(iter), iter_dir
 end
+-- This will probably get exposed in the api if that happens
+function project:files(inc_patt_arr, exc_patt_arr, dirname) -- iterate over the files in the project adhering to the provided patterns
```

Comment:
This should just get deleted as it's no longer used and there's not much point to having an iterator that just throws away values from another since they can easily be ignored.